### DISCUSSION TITLE
task failing issue in semaphore UI

### DISCUSSION THREAD

**Discussion by Tamilselvam9217** [semaphoreui/semaphore#2185] [Show and tell]
I have configured the Semaphore UI on my machine. I am encountering the following error while running tasks, but the same Ansible script is working properly via the CLI.
<img width="495" alt="semaphore_UI_issue" src="https://github.com/semaphoreui/semaphore/assets/147582739/7c8248c2-86bc-4ec6-b554-f6d25a60fc9e">

**Comment by fiftin** [comment]
It looks like host key checking issue.
